_sub/security/cloudtrail-alarm/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "cloud_watch_alarm_arn" {
```

Comment:
Minor thing. CloudWatch is name, so I think the output name ought to be `cloudwatch_alarm_arn`

Reply:
Yes, make sense. I am not sure why I put an underscore there.